games-arcade/ccleste/ccleste-1.4.0.recipe

```diff
@@ -0,0 +1,60 @@
+SUMMARY="A classic platform game"
+DESCRIPTION="Celeste Classic is the original free version of Celeste \
+, created in 2016. It was written for the Pico-8 virtual console. \
+The game can be found on Pico-8's website and on itch.io. \
+The original code for the game was written in a form of the Lua \
+programming language tailored to the Pico-8, although the version of \
+Celeste Classic found inside the full Celeste game is written in C#."
+
+HOMEPAGE="https://github.com/lemon32767/ccleste"
+COPYRIGHT="lemon32767"
+LICENSE=""
+REVISION="1"
+SOURCE_URI="https://github.com/lemon32767/ccleste/archive/refs/tags/v$portVersion.tar.gz"
+CHECKSUM_SHA256="32dfd797f3c863201e0c19aa97974c56a8ed589a34c0522503f25f6e1399edd6"
+SOURCE_DIR="ccleste-$portVersion"
+
+ARCHITECTURES="x86_64 x86"
+
+PROVIDES="
+ccleste$secondaryArchSuffix = $portVersion
+app:ccleste$secondaryArchSuffix = $portVersion
+cmd:ccleste
+"
```

Comment:
Change to:
```
PROVIDES="
	ccleste$secondaryArchSuffix = $portVersion
	app:Celeste$secondaryArchSuffix = $portVersion
	"
```
The window title mentions "Celeste", so this should reflect the name here.